test/mb_repl.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Open the node REPL then `require('path/to/mb_repl')`
+ * Used in QUnit tests.
+ */
+
+// Consistent environment stuff (mw, window/document via jsdom, jQuery)
+require(__dirname + '/mb_repl_head.js');
+
+// Load mw.config data
+require(__dirname + '/mw_config_data.js');
+
+require(__dirname + '/../morebits.js');
+global.Morebits = window.Morebits;
```

Comment:
Neat, I had heard of `globalThis` but didn't know `global` was a thing!

Reply:
Like 70% of the effort here was annoying `global`/`window`/`global.window` shit.  But global is the repl's speciality since there ain't no window